tools/generator/test/PBXTarget+ExtensionsTests.swift

```diff
@@ -0,0 +1,84 @@
+import XcodeProj
+import XCTest
+
+@testable import generator
+
+class PBXTargetExtensionsTests: XCTestCase {
+    let pbxTarget = PBXTarget(name: "chicken", productName: "MyChicken")
+
+    func test_buildableName_WithProductName() throws {
```

Comment:
Nit: can we use camel case?
```suggestion
    func test_buildableName_withProductName() throws {
```

Reply:
Done